SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -468,14 +520,14 @@ void USpatialReceiver::ReceiveActor(Worker_EntityId EntityId)
 
 			if (PendingAddComponent.EntityId == EntityId)
```

Comment:
With dynamic componenets there is the chance that there could be quite a lot of `PendingAddComponents` now right? Perhaps we should turn this into a multi-map using the EntityId as the key?

Reply:
Resolved offline